src/OrbitBase/ThreadUtilsWindows.cpp

```diff
@@ -12,6 +12,65 @@
 
 namespace orbit_base {
 
+static constexpr uint32_t kInvalidWindowsThreadId = 0;
+static constexpr uint32_t kInvalidWindowsProcessId_0 = 0;
+static constexpr uint32_t kInvalidWindowsProcessId_1 = 0xffffffff;
+
+namespace {
+
+// On Windows, thread and process ids are observed to be multiples of 4. Even though there is no
+// formal guarantee for this property, the current implementation of cross platform thread process
+// ids rely on it. https://devblogs.microsoft.com/oldnewthing/20080228-00/?p=23283
+inline [[nodiscard]] bool IsMultipleOfFour(uint32_t value) { return (value & 3) == 0; }
```

Comment:
constexpr?

Reply:
This takes in runtime values, we can't use constexpr right?

Reply:
I think you still can. It is enough that the function could be evaluated at compile time if the parameter is known.

Reply:
It's only used in this cpp file, will leave as is, on it's single line :-)